apiary-receiver/pom.xml

```diff
@@ -0,0 +1,18 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.expedia.apiary</groupId>
+    <artifactId>apiary-extensions-parent</artifactId>
+    <version>1.3.3-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>apiary-receiver</artifactId>
```

Comment:
I wonder if we should call the top-level module `apiary-receivers` (i.e. plural) like we did with `apiary-extensions`? Then this would become `apiary-receivers-parent` and the sub-modules could stay singular as each receiver is a single thing... What do others think? If we decide to stay with singular for all like it is now then I think this pom should just be renamed to `apiary-receiver-parent` to make it a bit clearer that it's used as a parent downstream.

Reply:
+1 for adding -parent
no strong preference for plural/singular, ok with either

Reply:
Adding -parent is a good idea. I agree with what you suggest for `apiary-receivers` and then sub-modules can be singular.

Reply:
Done